cilium-dbg/cmd/bpf_policy_get.go

```diff
@@ -199,6 +199,9 @@ func formatMap(w io.Writer, statsMap []policymap.PolicyEntryDump) {
 		} else if lbls := labelsID[id]; lbls != nil && len(lbls.Labels) > 0 {
 			first := true
 			for _, lbl := range lbls.Labels.GetPrintableModel() {
+				if lbl == "reserved:unknown" {
+					lbl = "ANY"
```

Comment:
Its quite minor, but might be worth considering noting the change in the release notes?

Reply:
Sure. I just started my contribution to the cilium codebase and don't know how to do that properly. Do you need me to remove the notes in the PR?

Reply:
Upgrade notes will be in: Documentation/operations/upgrade.rst. Right now we just cut a release so they're pretty empty but take a look at the file on one of the release branches for a good idea of the format.

> Do you need me to remove the notes in the PR?
Nope, just add some notes in the upgrade.rst doc

Reply:
ok. I will figure out the branch and open another PR for 1.18 branch.